File: src/main/scala/li/cil/oc/integration/enderio/ModEnderIO.java
package li.cil.oc.integration.enderio;

import li.cil.oc.api.Driver;
import li.cil.oc.integration.IMod;
import li.cil.oc.util.mods.Mods;

public final class ModEnderIO implements IMod {
    @Override
    public Mods.Mod getMod() {
        return Mods.EnderIO();
    }

    @Override
    public void initialize() {
        Driver.add(new DriverCapacitor());
    }
}

File: src/main/scala/li/cil/oc/integration/enderio/ModEnderIO.scala
package li.cil.oc.integration.enderio

import li.cil.oc.api
import li.cil.oc.integration.ModProxy
import li.cil.oc.integration.Mods

object ModEnderIO extends ModProxy {
  override def getMod = Mods.EnderIO

  override def initialize(): Unit = {
    api.IMC.registerWrenchTool("li.cil.oc.integration.enderio.EventHandlerEnderIO.useWrench")
    api.IMC.registerWrenchToolCheck("li.cil.oc.integration.enderio.EventHandlerEnderIO.isWrench")
  }
}
